### PR TITLE
god.

### PR DESCRIPTION
**Pull request by JimmyMcJuiceDrinker**
yes.

### CODE REVIEW

**Comment by JimmyMcJuiceDrinker** [comment]
YES

**Comment by JimmyMcJuiceDrinker** [comment]
YESSSSSSSSSSSSSSSSSSSS

**Comment by JimmyMcJuiceDrinker** [comment]
YES YES GHRD YES!

**Comment by JimmyMcJuiceDrinker** [comment]
SLAANESHAMS IS OVER!

**Comment by JimmyMcJuiceDrinker** [comment]
YES
YES YES YES YES

**Comment by JimmyMcJuiceDrinker** [comment]
BY ALL KNOWN LAWS OF AVIATION
THERE IS NO WAY A BEE SHOULD BE ABLE TO FLY
THEIR TINY WINGS SHOULDN'T BE ABLE
TO GET THEIR FAT LITTLE BODIES OFF THE GROUND

**Comment by JimmyMcJuiceDrinker** [comment]
BUT THE BEE OF COURSE FLIES ANYWAYS...
BECAUSE A BEE DOESN'T CARE WHAT A HUMAN THINKS
IS... IMPOSSIBLE

**Comment by JimmyMcJuiceDrinker** [comment]
BLACK

**Comment by JimmyMcJuiceDrinker** [comment]
YELLOW

**Comment by JimmyMcJuiceDrinker** [comment]
BLACK

**Comment by JimmyMcJuiceDrinker** [comment]
YELLOW

**Comment by JimmyMcJuiceDrinker** [comment]
BLACK

**Comment by JimmyMcJuiceDrinker** [comment]
YELLOW

**Comment by JimmyMcJuiceDrinker** [comment]
OOOOOOOOOOOOOOOOOH

**Comment by JimmyMcJuiceDrinker** [comment]
YELLOW

**Comment by JimmyMcJuiceDrinker** [comment]
BLACK

**Comment by JimmyMcJuiceDrinker** [comment]
LET'S SHAKE THINGS UP A LITTLE.

**Comment by JimmyMcJuiceDrinker** [comment]
.--.               .--.
           `.             .'    
           \          /      
    /\_     \ .-"-. /     _/\
   (         V ^ ^ V         )
   \_      _| 9_9 |_      _/
    `.    //\__o__/\\    .'
     `._//\=======/\\_.'
      /_/ /\=====/\ \_\
        _// \===/ \\_
        /_/_//`='\\_\_\   
            /_/     \_\

**Comment by MosiJ** [comment]
@JimmyMcJuiceDrinker K

**Comment by ChadTykonson** [comment]
@JimmyMcJuiceDrinker hi

**Comment by ChadTykonson** [comment]
@ @JimmyMcJuiceDrinker Hello JimmyMcJuiceDrinker

File: truth. [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Number 15:Burger King Foot Lettuce
+
+The last thing you'd want in your Burger King burger is someone's foot fungus.
+But as it turns out,that might be what you get.
+A 4channer uploaded a photo to the site anonymously to the site showcasing his feet in a plastic bin of lettuce with the statement
+"This is the lettuce you eat at Burger King".
+Admittedly he had shoes on but,that's even worse.
+The post went live at 11:38 PM on July 16 and a mere 20 minutes later,the Burger King in question was alerted to the rogue employee...
+at least,I hope he is rogue.How did it happen?
+Well the BK employee hadn't removed the Exif data from the uploaded photo,
```
Comment (civic-owner) [inline truth.:10]: when hondas fly

**Comment by JimmyMcJuiceDrinker** [comment]
A conflict has cometh comrades, we must combat it.

**Comment by JimmyMcJuiceDrinker** [comment]
Number 15: Burger king foot lettuce. The last thing you'd want in your Burger King burger is someone's foot fungus. But as it turns out, that might be what you get. A 4channer uploaded a photo anonymously to the site showcasing his feet in a plastic bin of lettuce. With the statement: "This is the lettuce you eat at Burger King." Admittedly, he had shoes on.

**Comment by JimmyMcJuiceDrinker** [comment]
But that's even worse.

The post went live at 11:38 PM on July 16, and a mere 20 minutes later, the Burger King in question was alerted to the rogue employee. At least, I hope he's rogue. How did it happen? Well, the BK employee hadn't removed the Exif data from the uploaded photo, which suggested the culprit was somewhere in Mayfield Heights, Ohio. This was at 11:47. Three minutes later at 11:50, the Burger King branch address was posted with wishes of happy

**Comment by JimmyMcJuiceDrinker** [comment]
unemployment. 5 minutes later, the news station was contacted by another 4channer. And three minutes later, at 11:58, a link was posted: BK's "Tell us about us" online forum. The foot photo, otherwise known as exhibit A, was attached. Cleveland Scene Magazine contacted the BK in question the next day. When questioned, the breakfast shift manager said "Oh, I know who that is. He's getting fired." Mystery solved, by 4chan. Now we can all go back to eating our fast food in peace.

File: truth. (hunk 1 above)
Comment (Vandonipugeuefhwihuefwoui) [reply truth.:10]: yes

**Comment by ghost** [comment]
Thank you for the pull request! I'm amazed I didn't notice this 27 days ago. Anyway, I'm afraid there's no place for your `truth` in our repo, but good work on discovering Pull Requests. I'd say to keep the PR's coming, but please... only useful contributions. Thanks again!

**Pull request closed without merging**